dashboard/app/models/blockly.rb

```diff
@@ -149,14 +149,35 @@ def blockly_options
     options = Rails.cache.fetch("#{cache_key}/blockly_level_options") do
       level = self
       # Use values from properties json when available (use String keys instead of Symbols for consistency)
-      level_prop = level.properties.dup || {}
+      level_prop = {}
```

Comment:
Would now be a good time to rename `level_prop` to something like e.g. `level_props`, `blockly_level` or `blockly_level_object`?